tpm2/test/tpm2_test.go

```diff
@@ -2080,3 +2080,29 @@ func TestDictionaryAttackLockReset(t *testing.T) {
 		t.Fatalf("got %d, expected 0", caps[0].(TaggedProperty).Value)
 	}
 }
+
+func TestTPMLDigest(t *testing.T) {
+	hashA := make([]byte, 20)
```

Comment:
I think this test might be more thorough if it put some sample data into hashA and hashB, such as
0102030405... and fffefdfcfe..., and then asserted the expected full serialization with the correct order of the correct sample hashes.

Reply:
agreed